### PR TITLE
updated the main config

### PR DESCRIPTION
**Pull request by VivekSinghDS**

### CODE REVIEW

File: toolkit.py [hunk 1]
```diff
@@ -74,14 +74,24 @@ def run_one_experiment(config: Config, config_path: str) -> None:
...
+         # TODO: Load results.csv
+        results_df = pd.read_csv(results_file_path)
+        prompts = results_df['prompt'].tolist()
+        ground_truths = results_df['ground_truth'].tolist()
+        model_preds = results_df['model_prediction'].tolist()
+        # TODO: Run Unit Tests
+        test_suite = LLMTestSuite(tests, prompts, ground_truths, model_preds)
+        # TODO: Save Unit Test Results
+        test_suite.save_test_results('unit_test_results.csv')
+        pass
```
Comment (benjaminye) [inline toolkit.py:93]:
```suggestion
```

**Review by benjaminye — CHANGES_REQUESTED** [review]
@VivekSinghDS, there has been a bit of change in the repo. See PR https://github.com/georgian-io/LLM-Finetuning-Hub/pull/118. We have migrated `toolkit.py` from the parent directory into `llmtune/cli/toolkit.py`.

To test your code, you just need to run (in a virtual environment) `pip install -e .` and do `llmtune [--config-path <your_config_path>]` where stuff in `[ ]` are optional.


In addition, we have adopted `ruff` as our linter and formatter. Please `pip install ruff` and run these before PR:
1. `ruff check --fix`
2. `ruff format`

File: toolkit.py [hunk 2]
```diff
@@ -74,14 +74,24 @@ def run_one_experiment(config: Config, config_path: str) -> None:
...
+        tests = TestRegistry.create_tests_from_list(llm_tests)
+         # TODO: Load results.csv
+        results_df = pd.read_csv(results_file_path)
+        prompts = results_df['prompt'].tolist()
+        ground_truths = results_df['ground_truth'].tolist()
+        model_preds = results_df['model_prediction'].tolist()
+        # TODO: Run Unit Tests
+        test_suite = LLMTestSuite(tests, prompts, ground_truths, model_preds)
+        # TODO: Save Unit Test Results
+        test_suite.save_test_results('unit_test_results.csv')
```
Comment (truskovskiyk) [inline toolkit.py:92]: could we remove hardcode here?

**Comment by benjaminye** [comment]
Closed as it is superseded by https://github.com/georgian-io/LLM-Finetuning-Toolkit/pull/131

**Pull request closed without merging**